Wright_Will/Assignments/ninja_gold/server.py

```diff
@@ -0,0 +1,28 @@
+from flask import Flask, render_template, request, redirect,session
+from random import randint
+app = Flask(__name__)
+app.secret_key = "fred"
+
+@app.route('/',methods=['GET','POST'])
```

Comment:
This assignment seems hard and daunting from the instructions, but looking at it whole (complete) ain't so bad. Thanks for helping your classmates with some hints on this one as well, it's appreciated.